src/transformers/modeling_utils.py

```diff
@@ -949,6 +949,11 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
 
             Passing :obj:`use_auth_token=True` is required when you want to use a private model.
 
+        .. note::
+
+            Use the special `"offline-mode" <https://huggingface.co/transformers/installation.html#offline-mode>`__ to
```

Comment:
```suggestion
            Activate the special `"offline-mode" <https://huggingface.co/transformers/installation.html#offline-mode>`__ to
```
(just to avoid the repetition of use)